Issue in type definitions (semicolon)

Hello there, I added `nodejs-snowflake` to a TypeScript project and it doesn't compile anymore due to the following error:

```
node_modules/nodejs-snowflake/nodejs_snowflake.d.ts(7,2): error TS1036: Statements are not allowed in ambient contexts.
```

This is due to the semicolon in line 7 of `nodejs_snowflake.d.ts`. If I manually remove this, my project compiles just fine.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -32,7 +32,7 @@ const CUSTOM_TS: &'static str = r#"
 export interface SnowflakeOpts {
     custom_epoch?: number; 
     instance_id?: number;
-};
+}
 "#;
 
 //// HELPER TYPES //////////////////////////////////////////////////////////////////////////

